test.js

```diff
@@ -1,4 +1,5 @@
 const tools = require('./fizzbuzz.js');
+const CustomError = require('./statusCodes.js');
```

Comment:
ditto here - variable naming for exports. Shouldn't be capitalized (unless you're exporting a class), and generally is the same name as the file unless another name would be clearer